### PR TITLE
Request skeleton

### PR DESCRIPTION
**Pull request by KirbyDD**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -7,10 +7,57 @@ app.use(express.json())
...
 
+app.get('/api/v1/projects', (request, response) => {
+  response.status(200).json('Initial Setup Complete for Projects');
+});
+
+app.get('/api/v1/projects/:id', (request, response) => {
+  response.status(200).json('Initial Setup Complete for Individual Project');
+});
+
+app.get('/api/v1/palattes', (request, response) => {
```
Comment (kaylalarson1990) [inline server.js]: palettes is spelled wrong  here! just fyi!

File: server.js [hunk 2]
```diff
@@ -7,10 +7,57 @@ app.use(express.json())
...
+  response.status(200).json('Initial Setup Complete for Individual Palette');
+});
+
+/* POST REQUESTS */
+
+app.post('/api/v1/projects', (request, response) => {
+  response.status(201).json('Initial Setup Complete for Adding Project');
+})
+
+app.post('/api/v1/projects', (request, response) => {
```
Comment (kaylalarson1990) [inline server.js]: this one should say palettes and not projects.

File: server.js [hunk 3]
```diff
@@ -7,10 +7,57 @@ app.use(express.json())
...
+  response.status(201).json('Initial Setup Complete for Adding Palette')
+})
+
+/* PATCH REQUESTS */
+
+app.patch('/api/v1/projects/:id', (request, response) => {
+  response.status(202).json('Initial Setup Complete for Updating Project');
+})
+
+app.patch('/api/v1/projects/:id', (request, response) => {
```
Comment (kaylalarson1990) [inline server.js]: this one should also say palettes!

File: server.js [hunk 4]
```diff
@@ -7,10 +7,57 @@ app.use(express.json())
...
+  response.status(202).json('Initial Setup Complete for Updating Palette')
+})
+
+/* DELETE REQUESTS */
+
+app.delete('/api/v1/projects/:id', (request, response) => {
+  response.status(204).json('Initial Setup Complete for Updating Project');
+})
+
+app.delete('/api/v1/projects/:id', (request, response) => {
```
Comment (kaylalarson1990) [inline server.js]: one more, palettes here too!

File: server.js [hunk 5]
```diff
@@ -7,10 +7,57 @@ app.use(express.json())
...
+  response.status(204).json('Initial Setup Complete for Updating Palette')
```
Comment (kaylalarson1990) [inline server.js]: thanks for adding the 204 for the delete!

File: server.js (hunk 1 above)
Comment (KirbyDD) [reply server.js]: Updated the spelling 👍

File: server.js (hunk 2 above)
Comment (KirbyDD) [reply server.js]: Corrected path to palettes

File: server.js (hunk 3 above)
Comment (KirbyDD) [reply server.js]: Corrected path to palettes

File: server.js (hunk 4 above)
Comment (KirbyDD) [reply server.js]: Corrected path to palettes

File: server.js (hunk 5 above)
Comment (KirbyDD) [reply server.js]: Corrected path to palettes

Comment (KirbyDD) [reply server.js]: 👍

**Pull request merged**